extensions/civetweb/tests/resources/generate.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
```

Comment:
The script is included for reproducibility and for easier understanding of how the certs were created (it could be deduced from looking at the PEM files, but that would take quite some time).
I agree that at least parts of this code (the functions) could be made into a generic tool (which this script can then include), but the certs generated here are specific to this test set.
However, I am not sure where we should place such code and whether we would really use it.
I think this might be worth revisiting if the need for test certs arise in the future.